spec/music_album_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'spec_helper'
+
+describe Album do
+  before(:each) do
+    album_info = %w[Horror Simon Source Label 2000-02-20]
+    @music_album = Album.new(album_info, true)
+  end
```

Comment:
- [ ] Great job working on unit tests! 💯 However, I'm getting errors on these ones regarding the music Album, please consider taking a look at how your class receives the parameters in the initialize method! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/222616146-9bfcd5e7-7a31-429b-88b6-cf570c4b3336.png)